ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -49,6 +49,30 @@ ScrollView {
                 }
             }
 
+            StyledText {
+                //% "Application Logs"
+                text: qsTrId("app-logs")
+                font.pixelSize: 15
```

Comment:
just add here:
`font.underline: mouseArea.containsMouse`

and remove onEntered/onExited handlers in below.
all done.

Reply:
Done.